src/components/variant/ui/change-variant-button/change-variant-button.tsx

```diff
@@ -0,0 +1,29 @@
+import styles from './change-variant-button.module.css'
+
+import { ButtonHTMLAttributes, ReactElement } from 'react'
+import cn from 'classnames'
+
+import { Icon } from '@ui/icon'
+
+interface IChangeVariantButtonProps
+  extends ButtonHTMLAttributes<HTMLButtonElement> {
+  direction: 'up' | 'down'
+}
+
+export const ChangeVariantButton = ({
+  className,
+  direction,
+  ...buttonProps
+}: IChangeVariantButtonProps): ReactElement => {
+  return (
+    <button
+      className={cn(styles.changeVariantButton, className)}
+      {...buttonProps}
+    >
+      <Icon
+        name="Arrow"
```

Comment:
In Figma layout in the UI Kit section there is icon set. I think we should use separate icons for arrow-down and arrow-up from this set instead of rotating.
https://www.figma.com/file/Tw3mb2qsQeyCnaMQ9DlD3h/Anfisa?node-id=3537%3A268395
<img width="817" alt="Screen Shot 2022-05-25 at 00 24 41" src="https://user-images.githubusercontent.com/96476703/170135419-491eb15b-0fb7-4572-b0e8-b38edcdb1e51.png">

Reply:
Yes, but it is one icon with rotation)

Reply:
It is, but if you need to apply some rotation you must add extra styles to it each time. While having a separate icon allows to just pass a correspondent name. The same regarding to smaller or larger versions of arrows -- you can't just scale it up or down with css, because scaling works in both dimensions and there will be thicker or thinner strokes than the original. So they need separate icons as well (it's not related to the case, just saying).

Reply:
Added ArrowButton component